_data/gaming.yml

```diff
@@ -104,6 +104,12 @@ websites:
       twitter: AskPlayStation
       img: psn.png
       tfa: No
+    
+    - name: Rockstar Games Social Club
+      url: http://socialclub.rockstargames.com
+      twitter: RockstarGames
```

Comment:
We usually use the support twitter account if available since most companies listen to that more frequently.
Please change the twitter handle to `RockstarSupport`